docs/developer_guide.md

```diff
@@ -90,6 +90,18 @@ The results will look somewhat like the following:
 opensearch-data-prepper   1.2.0-SNAPSHOT   3e81ef26250c   23 hours ago   566MB
 ```
 
+### Running smoke tests on a Docker Image
```

Comment:
This might be better in the `release/README.md` file. I think most developers probably don't need to run these tests.